restlicsharpclient/restliclientintegrationtest/RestClientIntegrationTests.cs

```diff
@@ -16,28 +16,34 @@
 
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
+using System;
+
 using com.linkedin.restli.test.api;
 using restlicsharpclient.restliclient;
 using restlicsharpclient.restliclient.request;
 using restlicsharpclient.restliclient.request.builder;
 using restlicsharpclient.restliclient.response;
-using restlicsharpclient.restliclient.response.decoder;
 
 namespace restlicsharpclient.restliclientintegrationtest
 {
     [TestClass]
-    public class RestClientTests
+    public class RestClientIntegrationTests
     {
         [TestMethod]
-        public void RestClient_GetGreeting()
+        public void RestClientIntegration_GetGreeting()
```

Comment:
No need to have prefix `RestClientIntegration_` here since they are under this named class. Or is this some C# test convention?